### ISSUE TITLE
Adding a new package/app - node_modules folder not being created

### ISSUE THREAD

**Issue by uriklar** [t3-oss/create-t3-turbo#49]
Hi! I've created a repo using this template. (Repo for reference: https://github.com/mumlatz/mumlatz)

I've tried adding a new app (tried package as well). Steps:  
*  Created a folder with a package.json file
*  adding some dependencies
* run `pnpm i`

Expected: node_modules folder should be created
Actual: node_modules folder isn't being created

When I remove the line
```
node-linker=hoisted
```

from the `.npmrc` file, the node_modules folder does get created. 

The strange thing is that even with this line there, when I delete the node_modules folder from an **pre-existing** app/package (IE - api, db, etc...) and re-run `pnpm i`, the node_modules folder will get created. 

So basically what i'm asking is: What am I missing in my wiring to get the new package to work the same as the existing ones?

Things i've tried:
Adding the package to `pnpm-workspace.yml` and running `pnpm i` 

Thanks!

**Comment by juliusmarminge** [comment]
It depends on what dependencies your package has, if they are hoisted to the root or kept in the package's dir.

What's the problem? Why you "need" the node modules at that level?

**Comment by uriklar** [comment]
Thanks for replying! Actually I see that the dependencies are working for me even when hoisted. So you're right, I don't need it. 

But isn't it strange that it works differently for all existing projects? (They have their node_modules in their directory and they get restored there after deleting/installing node_modules)

So I guess I can close the issue.
Thanks!
